package proto_ff;

import "yd_fieldoptions.proto";


message offlineoffline
{
	optional int32 lv = 1[(yd_fieldoptions.field_cname) = "角色等级"];
	optional int64 offlineExp = 2[(yd_fieldoptions.field_cname) = "离线基础经验"];
}


message Sheet_offlineoffline
{
	repeated offlineoffline offlineoffline_List = 1[(yd_fieldoptions.field_arysize)=1604];
}

message offlineitemlistitemDesc
{
	optional int32 Num = 1[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 Vle = 2[(yd_fieldoptions.field_cname) = "权重"];
	optional int64 ID = 3[(yd_fieldoptions.field_cname) = "ID"];
}

message offlineitemlist
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "序号"];
	optional int32 Lvdown = 2[(yd_fieldoptions.field_cname) = "等级下限"];
	optional int32 Lvup = 3[(yd_fieldoptions.field_cname) = "等级上限"];
	optional int32 daoju = 4[(yd_fieldoptions.field_cname) = "道具类型库"];
	repeated offlineitemlistitemDesc item = 5[(yd_fieldoptions.field_cname) = "道具", (yd_fieldoptions.field_arysize) = 22];
}


message Sheet_offlineitemlist
{
	repeated offlineitemlist offlineitemlist_List = 1[(yd_fieldoptions.field_arysize)=134];
}
